pages/common/fold.md

```diff
@@ -0,0 +1,15 @@
+# fold
+
+> Wraps each line in an input file to fit a specified width.
+
+- Wrap each line to default width (80):
+
+`fold {{file}}`
+
+- Wrap each line to width '30':
```

Comment:
minor suggestion: double quotation marks ("") are more common in regular writing (as opposed to code), so it would be more appropriate to use them.